lib/decompress/zstd_decompress_block.c

```diff
@@ -69,15 +69,56 @@ size_t ZSTD_getcBlockSize(const void* src, size_t srcSize,
     }
 }
 
+/* Allocate buffer for literals, either overlapping current dst, or split between dst and litExtraBuffer, or stored entirely within litExtraBuffer */
+static void ZSTD_allocateLiteralsBuffer(const streaming_operation streaming, void* const dst, const size_t dstCapacity, const size_t litSize,
```

Comment:
minor : parameters' order is surprising, it should respect coding conventions.

The general order is : 
1) state
2) mutable (out) parameters
3) input parameters

This would put `dctx` first, followed by `dst` & `dstCapacity`, then the rest.